packages/mantine/src/components/table/Table.types.ts

```diff
@@ -11,7 +10,7 @@ import {
 import {Dispatch, ReactElement, ReactNode, RefObject} from 'react';
 
 import {DateRangePickerValue} from '../date-range-picker/DateRangePickerInlineCalendar';
-import {TableLayoutProps} from './layouts/RowLayout.types'; // TODO https://coveord.atlassian.net/browse/ADUI-9182
+import {RowLayoutProps} from './layouts/RowLayout.types'; // TODO https://coveord.atlassian.net/browse/ADUI-9182
```

Comment:
```suggestion
import {RowLayoutProps} from './layouts/RowLayout.types'; 
```
Vu que tu fix la task en même temps